src/Utilities/GetOutput.hpp

```diff
@@ -6,13 +6,15 @@
 #include <sstream>
 #include <string>
 
+#include "Utilities/PrintHelpers.hpp"
+
 /*!
  * \ingroup UtilitiesGroup
  * \brief Get the streamed output of `t` as a `std::string`
  */
 template <typename T>
 std::string get_output(const T& t) {
   std::ostringstream os;
-  os << t;
+  print_value(os, t);
```

Comment:
Similar to the discussion in #4794, I don't like modifying the behavior of `get_output` because it is used in tests to emulate `operator<<`.  And if you do change it, update the documentation.